aio/content/examples/view-encapsulation/e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,56 @@
+import { browser, by, element, logging, WebElement } from 'protractor';
+
+describe('View Encapsulation App', () => {
+
+  beforeAll(() => browser.get(''));
+
+  it('should colour the top level `NoEncapsulationComponent` red', async () => {
+    const noEncapsulationHeading = element(by.css('app-root > app-no-encapsulation > h2'));
+    expect(await noEncapsulationHeading.getCssValue('color')).toEqual('rgba(255, 0, 0, 1)');
```

Comment:
Super minor thing just for readability, may be we can declare constants for colors:
```
const RED = 'rgba(255, 0, 0, 1)';
const GREEN = 'rgba(0, 128, 0, 1)';
const RED = 'rgba(255, 0, 0, 1)';
```
and later use them in tests:
```suggestion
    expect(await noEncapsulationHeading.getCssValue('color')).toEqual(RED);
```